lib/scripts/utils.sh

```diff
@@ -15,7 +15,7 @@ export AGENT_PIDFILE="${DATADOG_DIR}/run/agent.pid"
 export AGENT_CMD="./agent run --cfgpath dist/ --pidfile run/agent.pid"
 
 export TRACE_AGENT_PIDFILE="${DATADOG_DIR}/run/trace-agent.pid"
```

Comment:
nonblocking and unrelated to this PR, but I noticed we don't use `TRACE_AGENT_PIDFILE` and the other `PIDFILE` constants in as many places as we can

Reply:
Will handle that on a separate PR, thanks!